tools/protoxform/protoxform.py

```diff
@@ -0,0 +1,405 @@
+# protoc plugin to map from FileDescriptorProtos to a canonicaly formatted
```

Comment:
Everything in `api/` is generated by `protoxform`. The actual tooling deltas are in `tool/`.